setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+
+with open("README.md", "r") as desc:
+    long_description = desc.read()
+
+
+setup(
+	name="xtractmime",
+	version="1.0.0",
+	description="Package for sniffing MIME(Multipurpose Internet Mail Extension) types",
```

Comment:
I think “package” is redundant in the description of a Python package, and I don’t think it’s worth it spelling out MIME (to those familiar it won’t help, to those unfamiliar it may confuse because of the “mail” thing).

On the other hand, I think it’s important to mention https://mimesniff.spec.whatwg.org/ , since that’s what differentiates this package from any other Python package about MIME sniffing.

What about “Implementation of the MIME Sniffing standard”?

Reply:
> What about “Implementation of the MIME Sniffing standard”?

makes more sense, also adding URL to
 "Implementation of the MIME Sniffing standard (https://mimesniff.spec.whatwg.org/)"?